app/src/org/commcare/fragments/connect/ConnectDeliveryProgressDeliveryFragment.java

```diff
@@ -136,4 +136,10 @@ private HashMap<String, HashMap<String, Integer>> getStatusMap(ConnectJobRecord
         return statusMap;
     }
 
+
+    @Override
+    public void onResume() {
+        super.onResume();
+        binding.getRoot().requestLayout();
```

Comment:
@Jignesh-dimagi can you proide a technical description of the root cause in the PR description along with how this fixes this issue.